usr/sbin/registercloudguest

```diff
@@ -128,58 +129,94 @@ def register_modules(extensions, products, registered=[]):
         )
 
 # ----------------------------------------------------------------------------
-# Support custom config file with -f command line option
-config_file = None
-
-if '-f' in sys.argv:
-    idx = sys.argv.index('-f')
-    config_file = sys.argv[idx+1]
-
-force_new_registration = False
-if '--force-new' in sys.argv:
-    force_new_registration = True
-
-if '--delay' in sys.argv:
-    arg_index = sys.argv.index('--delay')
-    delay_time = sys.argv[arg_index + 1]
-    time.sleep(int(delay_time))
-
-user_smt_ip = None
-if '--smt-ip' in sys.argv:
-    arg_index = sys.argv.index('--smt-ip')
-    user_smt_ip = sys.argv[arg_index + 1]
-    # Validation has to wait for Python 3 implementation
-
-user_smt_fqdn = None
-if '--smt-fqdn' in sys.argv:
-    arg_index = sys.argv.index('--smt-fqdn')
-    user_smt_fqdn = sys.argv[arg_index + 1]
-
-user_smt_fp = None
-if '--smt-fp' in sys.argv:
-    arg_index = sys.argv.index('--smt-fp')
-    user_smt_fp = sys.argv[arg_index + 1]
-
-if user_smt_ip or user_smt_fqdn or user_smt_fp:
-    if not (user_smt_ip and user_smt_fqdn and user_smt_fp):
+argparse = argparse.ArgumentParser(description='Register on-demand instance')
+argparse.add_argument(
+    '--clean',
+    action='store_true',
+    dest='clean_up',
+    default=False,
+    help='Clean up registration data'
+)
+argparse.add_argument(
+    '-d', '--delay',
+    default=0,
+    dest='delay_time',
+    help='Delay the start of registartion by given value in seconds',
+    type=int
+)
+# default config file location not set it is encoded in the utils function
+# get_config(()
+argparse.add_argument(
+    '-f', '--config-file',
+    dest='config_file',
+    help='Path to config file, default: /etc/regionserverclnt.cfg',
+)
+argparse.add_argument(
+    '--force-new',
+    action='store_true',
+    dest='force_new_registration',
+    default=False,
+    help='Force a new registration, for excaptional cases only',
+)
+help_msg='The update server cert fingerprint. Use '
```

Comment:
The `--smt-fp` and `--smt-fqdn` arguments use the help message suffix ", use ..."; this one uses ". Use ..."

Reply:
????

Reply:
The help message for `--smt-fp` says "The update server cert fingerprint. Use in exceptional cases only"
The help message for `--smt-fqdn` says "Use a specific FQDN to connect to the update server, use in exceptional cases only"
The help message for `--smt-ip` says "Use a specific FQDN to connect to the update server, use in exceptional cases only"

I would like to see the same statement about exceptional use cases be used consistently in all cases. Either put a period in front of it, or a comma in front of it, in all cases, instead of mixing syntax.